q2_feature_classifier/classifier.py

```diff
@@ -226,9 +226,35 @@ def classify_sklearn(reads: DNAFASTAFormat, classifier: Pipeline,
     'reads_per_batch': Int % Range(0, None),
     'n_jobs': Int,
     'pre_dispatch': Str,
-    'confidence': Float,
-    'read_orientation': Str % Choices(['same', 'reverse-complement'])}
-
+    'confidence': Float % Range(
+        0, 1, inclusive_start=True, inclusive_end=True) | Str % Choices(
```

Comment:
`0` still feels a bit magical to me, but maybe with some additional explanation I can be steered in the right direction? Unless we have a reason for it, my vote is to move that to a `Str Choices` option.

Reply:
`0` is to match the sklearn API. If we introduce a `Calculate_but_do_not_use` option, it would just necessitate a couple extra lines of code to make `confidence=0`.

Reply:
Matching sklearn API sounds like a great reason to leave as-is - thanks for explaining!